src/Commands/Connection/InfoCommand.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Connection;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+use Terminus\Models\Environment;
+
+/**
+ * Class InfoCommand
+ * @package Pantheon\Terminus\Commands\Connection
+ */
+class InfoCommand extends TerminusCommand
+{
+    /**
+     * Retrieve connection info for a specific environment such as git, sftp, mysql, redis
+     *
+     * @authorized
+     *
+     * @command connection:info
+     *
+     * @param string $environment Name of the environment to retrieve
+     * @param string $filter Parameter filter (optional)
+     * @param array $options [fields=<env,param,value>] [format=<table|csv|yaml|json>]
+     *
+     * @return RowsOfFields
+     *
+     * @field-labels
+     *   env: Environment
+     *   param: Parameter
+     *   value: Connection Info
+     *
+     * @example connection:info awesome-site dev git_command --format=json
+     *   Display connection information only for the given parameter
+     *
+     */
+    public function connectionInfo(
+        $environment,
+        $filter = null,
+        $options = ['format' => 'table', 'fields' => 'param,value']
+    ) {
+        $site_env = explode('.', $environment);
```

Comment:
Yes. I was thinking about this too esp. when we're using defaults from `.env`
My first thought was to reason about argc but that falls apart as soon as there is more than one arg to pull  from local config. Not sure if `@site.env` future proofs this at all. Does that mean we might have `&organization`?